### ISSUE TITLE
Doc Accessibility ACC-018: Missing table captions

### ISSUE THREAD

**Issue by tfrancart** [sparna-git/shacl-play#331]
Add <caption> element inside property table element

**Comment by tfrancart** [comment]
```html
<table>
  <caption>Properties for {{libellé de la shape}}</caption>

</table>
``` 

- anglais : "Properties for "..." "
- français : "Propriétés de "..." "
